client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -45,6 +48,8 @@ export const DateUtils = {
   isAfter,
   isBefore,
   isEqual,
+  isValid,
+  age: (date: Date) => differenceInYears(Date.now(), date),
```

Comment:
When I enter an age the age always jumps back by 1 year.
I identified the problem in this age method, e.g. when entering the age 1 I get the following values in this age method:

```
now: Fri Jul 29 2022 09:13:41 GMT+1200 (New Zealand Standard Time),
dob: Thu Jul 29 2021 12:00:00 GMT+1200 (New Zealand Standard Time),
age: 0
```

This seems to work better:
```typescript
function removeTime(date: Date): Date {
  return new Date(date.getFullYear(), date.getMonth(), date.getDate());
}

...
age: (date: Date) => {
    const now = removeTime(new Date());
    const age = differenceInYears(now, removeTime(date));
    console.log(`now: ${now}, dob: ${removeTime(date)}, age: ${age}`);
    return age;
  },
```

which gives:
```
now: Fri Jul 29 2022 00:00:00 GMT+1200 (New Zealand Standard Time),
dob: Thu Jul 29 2021 00:00:00 GMT+1200 (New Zealand Standard Time),
age: 1
```

is there a better way to do removeTime?

Reply:
Well spotted! thanks for picking that one up!

The function you have is how I'd do it 😉 But then I checked the `date-fns` since we have that imported - they have a `startOfDay` and a `startOfToday` which fixes the problem too. I thought that startOfDay could be useful elsewhere..

Reply:
thanks! knew that date-fns must have something like this but could remember what to look for